src/main/resources/anniv5Resources/localization/eng/pixiepack/Powerstrings.json

```diff
@@ -0,0 +1,23 @@
+{
+  "${ModID}:FastHandsPower": {
+    "NAME": "Fast Hands",
+    "DESCRIPTIONS": [
+      "After playing a foreign card, draw ",
```

Comment:
You refer to different color everywhere but in powers, here you call 'em foreign. Please decide on one and use that everywhere.

Reply:
Missed this when I changed the keyword to match Prismatic, Fixed.